tests/templates/test_integration.py

```diff
@@ -54,7 +54,7 @@
         tf.enable_eager_execution()
 
     from tensorflow import Variable as TFVariable
-    INTERFACES.append(('tf', TFVariable))
+    INTERFACES.append(('tf', lambda x: TFVariable(x, dtype=tf.float64)))
```

Comment:
Oh, clever. Was that needed?

Reply:
Yep; by default, TensorFlow QNodes return double tensors, so TF will complain if the weights do not have a matching dtype